### PR TITLE
Update de_DE.lang

### PR DESCRIPTION
**Pull request by Fayti1703**
Also removed some duplicated lines in en_US.lang, namely:
* Pneumatic Chamber Wall
* Pneumatic Chamber Window
* Logistics Active Provider Frame

Will work on more in the future, I just don't have the time to now.

### CODE REVIEW

File: resources/assets/pneumaticcraft/lang/de_DE.lang [hunk 1]
```diff
@@ -72,24 +101,77 @@ item.turbineRotor.name=Turbinenrotor
 item.assemblyProgram.name=Montageprogramm
 item.emptyPCB.name=Leerer PCB
 item.unassembledPCB.name=Unbearbeiteter PCB
-item.pcbBlueprint.name=PCB Blaupause
-item.etchingAcidBucket.name=Eimer mit �tzender S�ure
+item.pcbBlauesprint.name=PCB Blaupause
```
Comment (wshadow) [inline resources/assets/pneumaticcraft/lang/de_DE.lang]: This looks wrong. Shouldn't this still be item.pcbBlueprint.name ?

Comment (MineMaarten) [inline resources/assets/pneumaticcraft/lang/de_DE.lang]: Nice find. I think you're right.

Comment (Fayti1703) [inline resources/assets/pneumaticcraft/lang/de_DE.lang]: Yeah... shouldn't have used Notepad++'s replace function as much as I did...

**Comment by Fayti1703** [comment]
Would you mind me closing this pull request and opening a new one later? I'm kinda having some problems with Git(Hub)...

**Comment by MineMaarten** [comment]
I've fixed it on my side already, locally. It will be in next version

**Pull request closed without merging**